Code/EnginePlugins/ParticlePlugin/System/ParticleSystemInstance.cpp

```diff
@@ -342,6 +342,7 @@ ezParticleSystemInstance::ezParticleSystemInstance()
 
 void ezParticleSystemInstance::Construct(ezUInt32 uiMaxParticles, ezWorld* pWorld, ezParticleEffectInstance* pOwnerEffect, float fSpawnCountMultiplier)
 {
+  m_BoundingVolume = ezBoundingBoxSphere::MakeInvalid();
```

Comment:
ezParticleSystemInstance's are reused, thus not resetting the variable would mean new instances used values from previous runs, so they might show up in the wrong location in the first few frames, and then deemed to be invisible, and then not updated...

Took me several hours to find this issue. It was mostly showing up with the gas cylinders, where the fire would not show up.